tsconfig.json

```diff
@@ -16,7 +17,8 @@
       {
         "name": "next"
       }
-    ]
+    ],
+    "target": "ES2017"
```

Comment:
what is this for? We normally use ES2021. Don't do anything you don't understand